scripts/dockerEVFBuild.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
i got this error when running `test 2`
![image](https://user-images.githubusercontent.com/3317835/101380162-6a8d3b80-3872-11eb-96a3-045569518585.png)

Reply:
I also encountered this.

In the container, you can see the file it's trying to copy over here: `/keg/app/node_modules/@keg-hub/tap-evf-sessions/node_modules/.bin`


It did not stop the build files from getting copied over, so it's not blocking for me.